### PR TITLE
Rwc2019 challenge receptionist

### PR DESCRIPTION
**Pull request by PetervDooren**

### CODE REVIEW

File: robot_smach_states/src/robot_smach_states/world_model.py [hunk 1]
```diff
@@ -220,6 +254,33 @@ def __init__(self, robot, entityDes, objectIDsDes=None, searchArea="on_top_of",
                                    transitions={'done': 'done'})
 
 
+class CheckVolumeEmpty(smach.StateMachine):
+    def __init__(self, robot, entity_des, volume="on_top_of", volume_threshold=0.0):
+        """ Constructor
+
+        :param robot: robot object
+        :param entity_des: EdEntityDesignator indicating the (furniture) object to check
+        :param volume: string defining of the entity to be checked, default = on_top_of
```
Comment (LarsJanssenTUe) [inline robot_smach_states/src/robot_smach_states/world_model.py]: unclear sentence, missing "part"?

File: robot_smach_states/src/robot_smach_states/world_model.py [hunk 2]
```diff
@@ -173,6 +173,40 @@ def execute(self, userdata=None):
...
         return 'done'
 
+
+class CheckEmpty(smach.State):
+    """
+    Check whether a volume of an entity is filled
+    """
+    def __init__(self, segmented_entity_ids_designator, entity_designator, volume, threshold=None):
+        """ Constructor
+        :param segmented_entity_ids_designator: designator containing the segmented objects
```
Comment (LarsJanssenTUe) [inline robot_smach_states/src/robot_smach_states/world_model.py]: "in the specified volume" adding

**Comment by PetervDooren** [comment]
The new smach state has been tested on the robot. The remaining bugs have been sorted out

File: robot_smach_states/src/robot_smach_states/world_model.py [hunk 3]
```diff
@@ -269,6 +305,33 @@ def __init__(self, robot, entityDes, objectIDsDes=None, searchArea="on_top_of",
                                    transitions={'done': 'done'})
 
 
+class CheckVolumeEmpty(smach.StateMachine):
+    def __init__(self, robot, entity_des, volume="on_top_of", volume_threshold=0.0):
+        """ Constructor
+
+        :param robot: robot object
+        :param entity_des: EdEntityDesignator indicating the (furniture) object to check
+        :param volume: string defining of the entity to be checked, default = on_top_of
```
Comment (PetervDooren) [inline robot_smach_states/src/robot_smach_states/world_model.py]:
```suggestion
        :param volume: string defining volume of the entity to be checked, default = on_top_of
```

File: robot_smach_states/src/robot_smach_states/world_model.py (hunk 1 above)
Comment (PetervDooren) [reply robot_smach_states/src/robot_smach_states/world_model.py]: changed to 'volume of the entity to be checked'

File: robot_smach_states/src/robot_smach_states/world_model.py (hunk 2 above)
Comment (PetervDooren) [reply robot_smach_states/src/robot_smach_states/world_model.py]: done

**Comment by MatthijsBurgh** [comment]
If you agree with my changes and CI of `challenge_receptionist` succeeds, you can merge this

**Pull request merged**